packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -3087,8 +3089,6 @@ function commitRoot(
     ReactSharedInternals.T = prevTransition;
     setCurrentUpdatePriority(previousUpdateLanePriority);
   }
-
-  return null;
```

Comment:
This is unused and mostly remove to have strict types so that `commitRoot` is assignable to  https://github.com/facebook/react/blob/0f1856c49febe96923e469f98c0b123130ea015c/packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js#L3292